core/src/Cabana_VerletList.hpp

```diff
@@ -295,9 +348,11 @@ struct VerletListBuilder
                             {
                                 // See if we should actually check this box for
                                 // neighbors.
-                                if ( linked_cell_list.cellStencil()
-                                         .grid.minDistanceToPoint(
-                                             x_p, y_p, z_p, i, j, k ) <= rsqr )
+                                if ( withinCutoff(
```

Comment:
@lebuller I don't think this was here when you reviewed. Trying to decide if we need more testing of this feature or not, but will merge if not

Reply:
It was not, just looked over it. Looks good to me. I'm not sure if this needs more testing or not. Not sure the right way to test this block more rigorously than we already do.